### ISSUE TITLE
Generic/MultipleStatementAlignment: false positive with non-scoped control structures

### ISSUE THREAD

**Issue by jrfnl** [squizlabs/PHP_CodeSniffer#2578]
Ported over from: https://github.com/squizlabs/PHP_CodeSniffer/issues/2529#issuecomment-502001484

```php
$phrase = 'phrase';
if ($a === true)
    if ($b === true)
        $phrase = 'another phrase';
else
    $phrase = 'different phrase';
```

Throws:
```
 3 | WARNING | [x] Equals sign not aligned with surrounding assignments; expected 9
   |         |     spaces but found 1 space
   |         |     (Generic.Formatting.MultipleStatementAlignment.NotSameWarning)
 8 | WARNING | [x] Equals sign not aligned with surrounding assignments; expected 5
   |         |     spaces but found 1 space
   |         |     (Generic.Formatting.MultipleStatementAlignment.NotSameWarning)
```

Caused by the control structure not having any braces.

**Comment by jrfnl** [comment]
Closing as replaced by PHPCSStandards/PHP_CodeSniffer#16
